src/Symfony/Component/Form/Resources/translations/validators.mn.xlf

```diff
@@ -14,6 +14,126 @@
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Үнэн зөв хэл сонгоно уу.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Үнэн зөв бүс сонгоно уу.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Үнэн зөв мөнгөний хэмжээ сонгоно уу.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Тоо оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>Нууц үг буруу байна.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Хувь утга оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Утга хоорондоо таарахгүй байна.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Үнэн зөв цаг оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Үнэн зөв цагын бүс оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Үнэн зөв URL оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Үнэн зөв хайх утга оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>Please provide a valid phone number.</source>
+                <target>Үнэн зөв утасны дугаар оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Сонгох хайрцаг буруу утгатай байна.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Үнэн зөв и-мэйл хаяг оруулна уу.</target>
+            </trans-unit>
+            <trans-unit id="126">
+                <source>Please select a valid option.</source>
+                <target>Үнэн зөв сонголт сонгоно уу.</target>
+            </trans-unit>
+            <trans-unit id="127">
+                <source>Please select a valid range.</source>
+                <target>Үнэн зөв хязгаарын утга сонгоно уу.</target>
```

Comment:
Can you verify that this is correct too?

Reply:
Yeah. This is correct

Reply:
Cool cool